tests/sentry/ratelimits/test_config.py

```diff
@@ -1,3 +1,4 @@
+from typing import Any
```

Comment:
fwiw adding `foo: Any` doesn't really help with the typing situation as it's the same as leaving it unannotated so generally you should find a more specific type for such variables